doc/permssions.md

```diff
@@ -0,0 +1,14 @@
+# Minimum permissions for collector account
```

Comment:
`Collector` is not the common name I think 
Maybe @avishayt @machacekondra can help here?

Reply:
Yeah, I wouldn't use the `collector` name as well. Maybe better is `agent` or something similar.